numba/tests/test_operators.py

```diff
@@ -491,6 +499,22 @@ def test_ne_scalar(self, flags=force_pyobj_flags):
     def test_ne_scalar_npm(self):
         self.test_ne_scalar(flags=Noflags)
 
+    def test_is_ellipsis(self):
+        cres = compile_isolated(self.op.is_usecase, (types.ellipsis, types.ellipsis))
+        cfunc = cres.entry_point
+        self.assertTrue(cfunc(Ellipsis, Ellipsis))
+
+    def test_is_void_ptr(self):
+        # can't call this directly from python, as void cannot be unboxed
+        cfunc_void = njit((types.voidptr, types.voidptr))(self.op.is_usecase)
```

Comment:
Think `njit` needs importing or this needs to be `@jit(nopyhon=True)` will appropriate line wrap.

Reply:
Any preference for which?

Reply:
Whichever you like :)